### DISCUSSION TITLE
device-generated-commands/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#53] [Announcements]
# device-generated-commands/

Big.

https://www.supergoodcode.com/device-generated-commands/

**Comment by rhysperry111** [comment]
> 3

Thank you Mike, as an official representative of Valve, for confirming that Half Life 3 is coming.

**Reply by zmike** [reply]
If you never hear from me again it's because your comment got me vanned.

**Reply by wertig0n** [reply]
Vanned... Is that like banned, only involves a van and... *shudder* duct tape?

**Comment by qwertychouskie** [comment]
> ## 3
> 
> Device. Generated. Commands.
> 
> Count ‘em.

Please forgive my doubts, Oh Mighty Chair

**Comment by sethdusek** [comment]
>While other development has been progressing, in the background I’ve been working on something big. Now, finally, I can talk about it.

Is there some sort of NDA with Vulkan extensions too? All the PRs seem to be created yesterday as well (after the extension was merged) despite some of the commits being 3 months old
